intelmq/tests/bots/parsers/sucuri/test_sucuri.data

```diff
@@ -0,0 +1 @@
ZD48YSBocmVmPSIvP3R5cGU9cmVkaXJlY3Rpb25zIj5yZWRpcmVjdGlvbnM8L2E+PC90ZD48dGQ+PHNwYW4+PGEgaHJlZj0iLz9kZXRhaWxzPW1hdGhhaWRzLmNvbSI+aHR0cDovL21hdGhhaWRzJiM0NjtjPCEtLSBzdWN1cmkgLS0+b20vP2Y8IS0tIHN1Y3VyaSAtLT5vbGlvPTlQPCEtLXN1Y3VyaS0tPk82WjNNVkY8L2E+PC9zcGFuPjwvdGQ+PC90cj4KPHRyPjx0ZD4yPC90ZD48dGQ+PGEgaHJlZj0iLz90eXBlPXJlZGlyZWN0aW9ucyI+cmVkaXJlY3Rpb25zPC9hPjwvdGQ+PHRkPjxzcGFuPjxhIGhyZWY9Ii8/ZGV0YWlscz1sdXh1cnl0ZHMuY29tIj5odHRwOi8vbHV4dXJ5dGRzJiM0Njs8IS0tIHN1Y3VyaSAtLT5jb20vZzwhLS0gc3VjdXJpIC0tPm8mIzQ2O3A8IS0tc3VjdXJpLS0+aHA/c2lkPTwvYT48L3NwYW4+PC90ZD48L3RyPgo8dHI+PHRkPjI8L3RkPjx0ZD48YSBocmVmPSIvP3R5cGU9cmVkaXJlY3Rpb25zIj5yZWRpcmVjdGlvbnM8L2E+PC90ZD48dGQ+PHNwYW4+PGEgaHJlZj0iLz9kZXRhaWxzPWx1Y2t5aGVyYnNzdXBwbHkuaW4iPmh0dHA6Ly9sdWNreTwhLS0gc3VjdXJpIC0tPmhlcmJzPCEtLSBzdWN1cmkgLS0+c3VwcGx5JjwhLS1zdWN1cmktLT4jNDY7aW4vPC9hPjwvc3Bhbj48L3RkPjwvdHI+Cjx0cj48dGQ+MjwvdGQ+PHRkPjxhIGhyZWY9Ii8/dHlwZT1yZWRpcmVjdGlvbnMiPnJlZGlyZWN0aW9uczwvYT48L3RkPjx0ZD48c3Bhbj48YSBocmVmPSIvP2RldGFpbHM9bGFhdG1pbnV0ZS5jb20iPmh0dHA6Ly9sYWF0bWludXRlJiM0NjtjPCEtLSBzdWN1cmkgLS0+b20vP2Y8IS0tIHN1Y3VyaSAtLT5vbGlvPTlQPCEtLXN1Y3VyaS0tPk82WjNNVkY8L2E+PC9zcGFuPjwvdGQ+PC90cj4KPHRyPjx0ZD4yPC90ZD48dGQ+PGEgaHJlZj0iLz90eXBlPXJlZGlyZWN0aW9ucyI+cmVkaXJlY3Rpb25zPC9hPjwvdGQ+PHRkPjxzcGFuPjxhIGhyZWY9Ii8/ZGV0YWlscz1odWF3ZWlkZXZpY2VzLmVzIj5odHRwOi8vaHVhd2VpZGV2aWNlcyYjNDY7PCEtLSBzdWN1cmkgLS0+ZXMvP2Y8IS0tIHN1Y3VyaSAtLT5vbGlvPTlQPCEtLXN1Y3VyaS0tPk9HRjZINEk8L2E+PC9zcGFuPjwvdGQ+PC90cj4KPHRyPjx0ZD4yPC90ZD48dGQ+PGEgaHJlZj0iLz90eXBlPXJlZGlyZWN0aW9ucyI+cmVkaXJlY3Rpb25zPC9hPjwvdGQ+PHRkPjxzcGFuPjxhIGhyZWY9Ii8/ZGV0YWlscz1ob3RtcDNzLmNvbSI+aHR0cDovL2hvdG1wM3MmIzQ2O2M8IS0tIHN1Y3VyaSAtLT5vbS8/ZjwhLS0gc3VjdXJpIC0tPm9saW89OVA8IS0tc3VjdXJpLS0+TzZaM01WRjwvYT48L3NwYW4+PC90ZD48L3RyPgo8dHI+PHRkPjI8L3RkPjx0ZD48YSBocmVmPSIvP3R5cGU9cmVkaXJlY3Rpb25zIj5yZWRpcmVjdGlvbnM8L2E+PC90ZD48dGQ+PHNwYW4+PGEgaHJlZj0iLz9kZXRhaWxzPWdvbGRwb2xlLmNvbSI+aHR0cDovL2dvbGRwb2xlJiM0NjtjPCEtLSBzdWN1cmkgLS0+b20vP2Y8IS0tIHN1Y3VyaSAtLT5vbGlvPTlQPCEtLXN1Y3VyaS0tPk82WjNNVkY8L2E+PC9zcGFuPjwvdGQ+PC90cj4KPHRyPjx0ZD4yPC90ZD48dGQ+PGEgaHJlZj0iLz90eXBlPXJlZGlyZWN0aW9ucyI+cmVkaXJlY3Rpb25zPC9hPjwvdGQ+PHRkPjxzcGFuPjxhIGhyZWY9Ii8/ZGV0YWlscz1kcmVzc2xpa2UuY29tIj5odHRwOi8vZHJlc3NsaWtlJiM0NjtjPCEtLSBzdWN1cmkgLS0+b20vP2Y8IS0tIHN1Y3VyaSAtLT5vbGlvPTlQPCEtLXN1Y3VyaS0tPk82WjNNVkY8L2E+PC9zcGFuPjwvdGQ+PC90cj4KPC90Ym9keT48dGZvb3Q+PHRyPjx0ZD48L3RkPjx0ZD48L3RkPjx0ZD48aT5MaW1pdGVkIHZpZXcgKDQwIHJvd3MpLi4uIE9ubHkgdGhlIHRvcCBlbnRyaWVzIGJlaW5nIGRpc3BsYXllZC48L2k+PC90ZD48L3RyPjwvdGZvb3Q+PC90YWJsZT48L3A+Cgo8aDQ+RW5jb2RlZCBqYXZhc2NyaXB0PC9oND4KPHA+CkVuY29kZWQgamF2YXNjcmlwdCAocmVkaXJlY3RpbmcgdG8gYmxhY2tob2xlIGFuZCBvdGhlciBleHBsb2l0IGtpdHMpIG9yIHRvIGJ1aWxkCmEgcmVtb3RlIGNhbGwuCjx0YWJsZSBjbGFzcz0iZ3B0YWJsZSBtd3RhYmxlIHRhYmxlIj48dGhlYWQ+PHRyPjx0aD4jIG9mIHNpdGVzIGluZmVjdGVkPC90aD48dGg+VHlwZTwvdGg+PHRoPk1hbHdhcmUgLyBEb21haW5zPC90aD48L3RyPjwvdGhlYWQ+PHRib2R5Pjx0cj48dGQ+MTI8L3RkPjx0ZD48YSBocmVmPSIvP3R5cGU9amF2YXNjcmlwdCI+amF2YXNjcmlwdDwvYT48L3RkPjx0ZD48c3Bhbj48YSBocmVmPSIvP2RldGFpbHM9ZGl2LWNsYXNzLWNvbnRhaW5l
```

Comment:
Please add this file decoded - human readable and only encoded it in the test when it is needed. Makes it a lot easier for humans to inspect the data format.

Reply:
I don't understand exactly. Should i have in intelmq/tests/bots/parsers/sucuri/ two data files? First encoded file for test and second decoded file for easier inspekt? Or should i have there only decoded file and encoded data should be in test_sucuri.py?

Reply:
Just save the plain HTML here. When reading the file you can encode it on the fly, e.g. https://github.com/certtools/intelmq/blob/develop/intelmq/tests/bots/parsers/generic/test_parser_csv.py#L16